### ISSUE TITLE
Support default values for collections.namedtuple?

### ISSUE THREAD

**Issue by ilevkivskyi** [python/mypy#4788]
Python 3.7 adds support for specifying default values in call to `collections.namedtuple`, see https://github.com/python/cpython/pull/4859. So for example this is valid:
```python
from collections import namedtuple

NT = namedtuple('NT', ('x', 'y'), defaults=(0, 0))
NT()
NT(1)
NT(1, 2)
```
Should we support this, or supporting default values for `typing.NamedTuple` (already implemented) is enough?

**Comment by gvanrossum** [comment]
The short form of `collections.namedtuple` is very popular (ask @rhettinger) and I think we should make it as easy as possible to translate its use to `typing.NamedTuple`. So I think we should support it.

**Comment by JukkaL** [comment]
I agree with @gvanrossum. Also, this doesn't look like a lot of work to implement. Tagging as high priority since 3.7 release date is pretty close.

**Comment by JelleZijlstra** [comment]
Fixed by #5215.
